Jenkinsfile_CNP

```diff
@@ -6,7 +6,7 @@ def channel = '#rpts-builds'
 def type = "java"
 def product = "rpts"
 def component = "api"
-def branchesToSync = ['demo', 'perftest', 'ithc']
+def branchesToSync = ['demo']
```

Comment:
you will need it to sync otherwise these changes in this PR won't get added to those branches and then it won't remove the infra in the environments we are removing